site/content/entry/yuzu-mini-dev-2/index.md

```diff
@@ -46,27 +45,27 @@ What got me into emulation mainly was, yuzu's very [first video](https://www.you
 **Q: That's really interesting to hear. Before yuzu and Switch emulation, have you ever worked on other older emulators?**
 
 **R:** No, I've never worked on a non-Switch emulator. The closest thing I did related to emulation was a clone of Pico-8 Celeste for the NES in 6502 assembly.
-It was a fun experience because I learnt the curse and blessing that it is, working that close to the hardware.
-And it showed me what a pain it was, to program classics like Super Mario Bros. 3.
+It was a fun experience, because I learnt the curse, and blessing, that it is, working that close to the hardware; it showed me what a pain it was to program classics like Super Mario Bros. 3.
 
 
-**Q: Let's get a bit technical. What areas of switch emulation have you worked on mainly and why? Which other areas keep you interested?**
+**Q: Let's get a bit technical. What areas of Switch emulation have you worked on the most, and why? Which other areas keep you interested?**
 
-**R:** I've worked on the graphics department and the compute department of Switch emulation.
-For those that don't know, the Switch's GPU - it's a Nvidia Tegra X1, the same architecture of a GTX 960+.
-Emulating a modern GPU (Tegra) by using other modern GPUs (user end), is my area of interest. 
+**R:** I've worked on the graphics, and compute departments of Switch emulation.
+Emulating a modern GPU (Tegra) by using other modern GPUs (user end) is my area of interest.
+Here's a cool fact for those that don't know: the Switch's GPU - it's a Nvidia Tegra X1, and it utilizes the same architecture of a GTX 960+.
```

Comment:
Should be: `Here's a cool fact for those who don't know: the Switch's GPU is an Nvidia Tegra X1, and it utilizes the same architecture of a GTX 960+.`